ci-operator/step-registry/ipi/conf/tang/ipi-conf-tang-commands.sh

```diff
@@ -39,7 +39,7 @@ echo "butane_version_list:" "${butane_version_list[@]}"
 TANG_SERVER_KEY=$(ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${SSH_PRIV_KEY_PATH}" ${BASTION_SSH_USER}@${BASTION_IP} "sudo podman exec -it tang tang-show-keys 8080")
 
 declare -a roles=("master" "worker")
-ret_code=0
+ret_code=1
 for butane_version in "${butane_version_list[@]}"; do
   for role in "${roles[@]}"; do
```

Comment:
I guess need to move code `ret_code=0`after line43. 
When checking 4.13.0 failed, reset  `ret_code=0` and continue to use 4.12.0 bu file to generate manifests file.

Reply:
I set ret_code to 1 by default, just to be more restrict.

Reply:
Yeah, your proposal should be able to fix this issue, too, for easy understanding the code, I disabled `errexit` as workaround.

Reply:
got it, thanks for fixing.